openfisca_core/formulas.py

```diff
@@ -532,8 +532,22 @@ def exec_function(self, simulation, period, *extra_params):
 
     def graph_parameters(self, edges, get_input_variables_and_parameters, nodes, visited):
         """Recursively build a graph of formulas."""
-        for dated_formula in self.dated_formulas:
-            dated_formula['formula'].graph_parameters(edges, get_input_variables_and_parameters, nodes, visited)
+        if self.dated_formulas is not None:
+            for dated_formula in self.dated_formulas:
+                dated_formula['formula'].graph_parameters(edges, get_input_variables_and_parameters, nodes, visited)
+        else:
+            holder = self.holder
```

Comment:
This code was wrongfully removed in `14.0.0`, it just copy pasted back, so please don't me questions about it 😨